### PR TITLE
Move Authlog to Laravel

### PR DESCRIPTION
**Pull request by Jellyfrog**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -212,4 +213,13 @@ protected function updateDashboard(User $user, $dashboard)
 
         return false;
     }
+
+    public function authlog()
+    {
+        $this->authorize('manage', User::class);
```
[the remark this answers is not in the document]
Comment (Jellyfrog) [reply app/Http/Controllers/UserController.php]: I think it's fine, if you can manage users, you should be able to view this

[the remark this answers is not in the document]
Comment (murrant) [reply app/Http/Controllers/UserController.php]: Correct, but what if you want to delegate access to the authlog to someone who shouldn't manage users? We'll leave it for now. Thanks.

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -212,4 +213,13 @@ protected function updateDashboard(User $user, $dashboard)
...
+
+        return view('user.authlog', [
+            'authlog' => AuthLog::orderBy('datetime', 'DESC')->get(),
```
Comment (murrant) [inline app/Http/Controllers/UserController.php]: Should we limit the results or just assume daily.sh keeps them from being ridiculous?

Comment (Jellyfrog) [reply app/Http/Controllers/UserController.php]: I think it's fine since it's cleant in daily

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-56-release-changelog-september-2019/9671/1

**Pull request merged**